nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-audit/src/test/java/org/nuxeo/elasticsearch/TestAuditWithElasticSearch.java

```diff
@@ -107,31 +110,31 @@ public void shouldLogInAudit() throws Exception {
         List<LogEntry> trail = reader.getLogEntriesFor(doc.getId(), doc.getRepositoryName());
 
         Assert.assertNotNull(trail);
```

Comment:
You could do a static import on `assertNotNull` also while you're at it... (although all this should probably be in a cleanup commit)

Reply:
Sure, forgot the good practices.